specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/alerts.json

```diff
@@ -0,0 +1,1022 @@
+              }
+            ]
+          },
+          "description": "The kill chain related intent behind the alert. For list of supported values, and explanations of Azure Security Center's supported kill chain intents."
+        },
+        "startTimeUtc": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC time of the first event or activity included in the alert in ISO8601 format."
+        },
+        "endTimeUtc": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC time of the last event or activity included in the alert in ISO8601 format."
+        },
+        "resourceIdentifiers": {
+          "readOnly": true,
+          "type": "array",
+          "description": "The resource identifiers for this alert which can be used to direct the alert to the right product exposure group (tenant, workspace, subscription etc.). There can be multiple identifiers of different type per alert.",
+          "items": {
+            "$ref": "#/definitions/ResourceIdentifier"
+          }
+        },
+        "remediationSteps": {
+          "readOnly": true,
+          "type": "array",
+          "description": "Manual action items to take to remediate the alert.",
+          "items": {
+            "readOnly": true,
+            "type": "string"
+          }
+        },
+        "vendorName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the vendor that raises the alert."
+        },
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Active",
+            "Resolved",
+            "Dismissed"
+          ],
+          "x-ms-enum": {
+            "name": "alertStatus",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Active",
+                "description": "An alert which doesn't specify a value is assigned the status 'Active'"
+              },
+              {
+                "value": "Resolved",
+                "description": "Alert closed after handling"
+              },
+              {
+                "value": "Dismissed",
+                "description": "Alert dismissed as false positive"
+              }
+            ]
+          },
+          "description": "The life cycle status of the alert. Supported statuses are: Active, Resolved, Dismissed."
+        },
+        "extendedLinks": {
+          "readOnly": true,
+          "type": "array",
+          "description": "Links related to the alert",
+          "items": {
+            "$ref": "#/definitions/AlertExtendedLinks"
+          }
+        },
+        "alertUri": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A direct link to the alert page in Azure Portal."
+        },
+        "timeGenerated": {
```

Comment:
There are other fields named `startTimeUtc` & `endTimeUtc`. Please could you name all your date-time fields consistently (either with the `Utc` suffix, or without it)?

Reply:
Done